Makefile

```diff
@@ -15,6 +15,7 @@ install-python: build-python
 
 install-go:
 	(cd ./go && go install bystro/cmd/dosage)
```

Comment:
Because we install here, we need to further down call the binary in the user's path, and not something in the go/cmd/** directories. This also requires us to use shell=True in subprocess.Popen; without that the environment isn't loaded and the binaries are inaccessible